### DISCUSSION TITLE
[ How can I transfer classes in flutter to flet?]

### DISCUSSION THREAD

**Discussion by burhansvural** [flet-dev/flet#2503] [Q&A] [answered]
### Question

I want to start a study to transfer all classes in Flutter to Flet. Is there a document for the roadmap? Also, is there any documentation on how to use the graphics drawing engine and how to draw the controls? How can I transfer classes in flutter to flet? Is there an example explanation? Help me with this?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
I started a series on that: https://youtube.com/playlist?list=PLHYEoLmkidc1YrgREPrJvhr_JEv00EFP8&feature=shared

**Reply by burhansvural** [reply]
thanks @ndonkoHenri 
Is there a professionally written document?
How can I use advanced control plotting techniques? How are control rendering operations done in Flet?
